app/domain/invoices/abacus/course_invoice.rb

```diff
@@ -12,34 +12,65 @@ class CourseInvoice
 
       def initialize(participation)
         @participation = participation
+        @course = participation.event
       end
 
       def positions
+        grouping = "#{Event::Course.human_attribute_name(@participation.price_category) if @participation.price_category?} - #{@course.kind.level}"
```

Comment:
Soll das nicht mehr via I18n übersetzbar sein? Generell verwirrt mich diese String concatination etwas :thinking:

Reply:
was gibt es den hier zu übersetzen? ein grouping ist zb. `Normalpreis - Einstiegskurs`. soll ich das in i18n als `grouping: "%{price_category} - %{level}` hinzufügen, falls man den separator (-) ändern möchte?

Reply:
Ah my bad ich habs optisch verglichen mit dem bisherigen wo ein Call zu `I18n.t` gemacht wird aber das hast du ja hier einfach (besser) via `human_attribute_name` gelöst. Das passt so :)